javascript/helpers.js

```diff
@@ -0,0 +1,32 @@
+import axios from 'axios'
+
+export const waitForJob = async (jobId, jobsBaseUrl, API_TOKEN) => {
```

Comment:
There is a 'PENDING' job state as well. This should take into account all of the job's states.